lib/jacoco/plugin.rb

```diff
@@ -32,46 +41,78 @@ def parse(path)
     # blah/blah/java/slashed_package/Source.java
     #
     def report(path, delimiter = '/java/')
```

Comment:
@antonyalkmim since kotlin is now supported delimiter should also be changed. Right now your setup will work only for projects with kotlin sources in java folder. I'd change this to something like `/\/java\/|\/kotlin\//`.

Reply:
You're right! Even if we can set the delimiter from `Dangerfile` like this:
 ```ruby
jacoco.report "path/to/jacoco.xml", 'kotlin'
```
its nicer when the lib do the trick for us.